app/_src/gateway/kong-manager/auth/workspaces-and-teams.md

```diff
@@ -41,29 +41,30 @@ For this example, start by creating a simple workspace called `SecureWorkspace`.
 2. On the workspaces tab, click on **New Workspace**.
 3. Create a workspace named `SecureWorkspace` and select a color or image for the workspace avatar.
 
-    Each workspace name should be unique, regardless of letter case. For example, naming one workspace “Payments” and another one “payments” will create two different workspaces that appear identical.
-
     {:.important}
-    > Do not give a workspace the same name as any of these major routes in Kong Manager:
+    > Workspace names are **case sensitive** ("Payments" and "payments" are not equal), so it is recommended you give your worksapces unique names regardless of letter case to prevent confusion.
```

Comment:
```suggestion
    > Workspace names are **case sensitive** ("Payments" and "payments" are not equal), so it is recommended you give your workspaces unique names regardless of letter case to prevent confusion.
```